configure.php

```diff
@@ -1,44 +1,33 @@
 <?php
   $dir = getcwd();
   echo $dir."\n";
-  
-  $targets = array( "local", "dev", "live", "new" ) ;
-  
+
+  $targets = array( "dev", "live" );
+
   if ( ( sizeof( $argv ) < 3 ) ||
        !in_array( strtolower( $argv[ 1 ] ), $targets ) )  {
-    echo "Usage: php configure.php (dev|live|local) (configuration location)\n -v";
+    echo "Usage: php configure.php (dev|live) (configuration directory location)\n -v";
     exit;
   }
 
   $target = strtolower( $argv[ 1 ] );
-  $config = $argv[ 2 ];
-  $verbose = ( ( sizeof( $argv ) > 2 ) && ( $argv[ 2 ] == '-v' ) );
+  $configDir = $argv[ 2 ];
+  $verbose = ( ( sizeof( $argv ) > 3 ) && ( $argv[ 3 ] == '-v' ) );
 
   $files = array(
-    "reg-api/.htaccess",
```

Comment:
There are no differences between the `dev` and `live` version of this file so renamed one to `.htaccess` and removed the other.